src/__tests__/doFetchArgs.test.tsx

```diff
@@ -61,15 +60,15 @@ describe('doFetchArgs: general usages', (): void => {
       HTTPMethod.POST,
       controller,
       '/route',
-      {},
+      {}
     )
     expect(url).toBe('https://example.com/path/route')
   })
 
   it('should correctly modify the options with the request interceptor', async (): Promise<void> => {
     const controller = new AbortController()
     const interceptors = {
-      request(options: any) {
+      request (options: any) {
```

Comment:
I find this rule kind of weird.. Space before function parentheses.

Reply:
I don't like it either. Can we remove?